pgactivity/cli.py

```diff
@@ -97,6 +97,16 @@ def get_parser() -> ArgumentParser:
         metavar="FIELD:REGEX",
         default=[],
     )
+    # --refresh
+    parser.add_argument(
+        "--refresh",
+        dest="refresh",
+        help=("Refresh frequency."),
```

Comment:
Superfluous parentheses.

Reply:
Also, in man page it's called "Refresh rate", not "frequency"; perhaps make the naming uniform (slightly preferring "rate" over "frequency")?

Reply:
I applied your proposals and amended the previous patch.

Then I went into a uniformisation spree in a separate commit. (should probably be a different PR ?)